coldfront/plugins/qumulo/tasks.py

```diff
@@ -156,17 +157,17 @@ def __ad_users_and_handle_errors(
     group_name = acl_allocation.get_attribute("storage_acl_name")
 
     if len(good_keys) > 0:
-        user_dns = [user["dn"] for user in good_keys]
+        member_dns = [user["dn"] for user in good_keys]
         try:
-            active_directory_api.add_user_dns_to_ad_group(user_dns, group_name)
+            active_directory_api.add_members_to_ad_group(member_dns, group_name)
         except Exception as e:
             logger.error(f"Error adding users to AD group: {e}")
             __send_error_adding_users_email(acl_allocation, wustlkeys)
             return
 
-        for user in good_keys:
+        for member in good_keys:
             AclAllocations.add_user_to_access_allocation(
-                user["wustlkey"], acl_allocation
+                member["wustlkey"], acl_allocation, member["is_group"]
             )
     if len(bad_keys) > 0:
         __send_invalid_users_email(acl_allocation, bad_keys)
```

Comment:
As noted, right now it won't cause any errors, because groups/group-users don't have emails.  Will add a follow up ticket to update this to extract emails from group users if needed.

I think, in general though, we probably won't be emailing every single user of a group.  One case I could see is for something like User Support being a group that is added as a group...we could then give that "User" the user support email instead of doing all the extra work that that email is ostensibly doing already